style.css

```diff
@@ -0,0 +1,234 @@
+.body {
+    background-color: #f8fafc;
+    margin: 0 auto;
+}
+
+.header {
+    background-color: white;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 100px;
+    font-family: 'Roboto', sans-serif;
+    letter-spacing: 2px;
+    padding: 0px 0px 0px 200px;
+    box-shadow: 0 5px 8px -2px lightgray;
+}
+.pavadinimas {
+    float: left;
+}
+span {
+    color: #0000ff;
+}
+.fa-brands {
+    color: blue;
+    padding: 0px 10px;
+}
+.icons {
+    padding: 0px 200px 0px 40px;
```

Comment:
1. If you would want to position element this way - you should use margin instead of padding 
2. You should not position these elements this way at all. Please style the container accordingly to position these elements instead of fixed values for the children elements.